Model/Config.php

```diff
@@ -188,7 +191,17 @@ public function getCmsBlocks()
      */
     private function initialize()
     {
-        $data = $this->reader->read();
+        $data               = $this->reader->read();
         $this->loadedConfig = $data;
     }
+
+    /**
+     * Method for retrieving dynamic config shipping path
+     *
+     * @return string
+     */
+    public function getDynamicShippingConfigPath(): string
+    {
+        return self::CONFIG_PATH_DYNAMIC_SHIPPING_TAX_CLASS;
```

Comment:
I do not think this adds much value. I think we should add a new method in `\FireGento\MageSetup\Model\System\Config`, which returns the actual configuration value of that option.